base/applications/cmdutils/taskkill/lang/zh-CN.rc

```diff
@@ -1,4 +1,7 @@
-/* Simplified Chinese translation by Henry Tang Ih 2016 (henrytang2@hotmail.com) */
+/* Simplified Chinese translation by 
```

Comment:
Why?
Also, please make sure your editor does not auto-format, auto-add line-endings and insert a BOM before.

Reply:
Because I translated two new sentences in to Chinese.
so I add my name below the origin translator's name In case there's anything wrong In my translation

Reply:
> Why?
> Also, please make sure your editor does not auto-format, auto-add line-endings and insert a BOM before.

I've checked. No BOM and no anything strange. previous problem is caused by Tabs